README.mdown

```diff
@@ -166,6 +167,37 @@ supports the "groupPrefix" parameter. This will add a prefix to the Ganglia metr
 The Zabbix reporter requires the dependency `zabbix-sender-0.0.3.jar` (Maven group ID: io.github.hengyunabc,
 Maven artifact ID: zabbix-sender, version: 0.0.3).
 
+### InfluxDB
+
+The InfluxDB reporter currently requires the dependency [`metrics-influxdb-1.1.6.jar`](http://repo1.maven.org/maven2/com/izettle/metrics-influxdb/1.1.6/metrics-influxdb-1.1.6.jar)
+[Sources](https://github.com/iZettle/dropwizard-metrics-influxdb) (Maven group ID: com.izettle,
+Maven artifact ID: metrics-influxdb, version 1.1.6).
+
+Dropwizard 4.0 will introduce native InfluxDB support.
+
+InfluxDB connection requires a few more configuration properties:
+
+    influxdb:
+      -
+        period: 60
+        timeunit: 'SECONDS'
+        protocol: 'http'
+        auth: 'troll:admin123'
+        dbName: 'db1'
+        connectionTimeout: 1000
+        readTimeout: 1000
+        tag: 'node1'
```

Comment:
Hi,
Hmm, yes, ` dropwizard-metrics-influxdb` has a `tags` map. Here I have put this `tag: 'node1'` as value for `host` tag (https://github.com/addthis/metrics-reporter-config/pull/29/files#diff-684e2b30b130fdd0b48aace8d4a5ca48R35). I wanted this `tag` to be more like `prefix` configuration key used for e.g. graphite (and I've reused its logic to resolve `host.name` variable etc) - to distinct metrics between hosts. So I see two options here:
1. `tag` -> `tags`, but then also this configuration property value has to become a full map
2. or rename this `tag` to e.g. `tag-host`
What do you think @jhorwit2 ?